local.conf.sample

```diff
@@ -0,0 +1,69 @@
+[[local|localrc]]
+
+enable_plugin os-xenapi https://github.com/openstack/os-xenapi.git
+
+###########################################################################
+# Common for Controller and Computer Nodes
+
+# OpenStack VM settings
+GUEST_NAME=
+
+# Compute settings
+VNCSERVER_PROXYCLIENT_ADDRESS=
+XENAPI_PASSWORD=
+
+# DevStack settings
+ENABLED_SERVICES+=neutron,q-domua
+###########################################################################
+
+
+###########################################################################
+# Only for Compute Nodes
+
+# Controller node settings
+#SERVICE_HOST=CONTROLLER_IP
```

Comment:
Instead of CONTROLLER_IP always use something like @CONTROLLER_IP@ for replacements